### ISSUE TITLE
inArea error

### ISSUE THREAD

**Issue by AlexTriada** [A3Antistasi/A3-Antistasi#185]
https://github.com/A3Antistasi/A3-Antistasi/blob/f58f7e40e1fc860bef0f1c177d639219737e3158/A3-Antistasi/AI/defendDrillAI.sqf#L73

_edificios = _edificios select {((_x buildingPos -1) isEqualTo []) and !((typeof _bld) in UPSMON_Bld_remove) and (_x **inAera** _marcador)};

must be 

_edificios = _edificios select {((_x buildingPos -1) isEqualTo []) and !((typeof _bld) in UPSMON_Bld_remove) and (_x **inArea** _marcador)};

**Comment by AlexTriada** [comment]
again I found a bug with sqflint

**Comment by AlexTriada** [comment]
vs code:
https://code.visualstudio.com/
sqflint:
https://marketplace.visualstudio.com/items?itemName=skacekachna.sqflint

**Comment by AlexTriada** [comment]
sqflint count 3922 problems))) looks like I'll have to create 3922 bugs

**Comment by Barbolani77** [comment]
Thanks! That script is not executed, just under development

**Comment by Barbolani77** [comment]
Btw that tool results must be wrong. If Antistasi had that number of errors It wouldnt even initialise. Anyway let me know if you check It And notice something as critical as the one you spotted

**Comment by AlexTriada** [comment]
OK

**Comment by SkaceKamen** [comment]
I'm the creator of the sqflint ... 90% of the errors are false positives, caused by using editor placed objects (which I fixed in new version) and other stuff, the extension is not perfect. But there are some actual errors like the one mentioned here, which can cause issues under centrain conditions.

I actually cloned this repo two days ago and I'm going through the errors manually to discover the actual issues.
